### PR TITLE
add pr template.

### PR DESCRIPTION
**Pull request by haiker2011**
Signed-off-by: sunhaizhou <sunhaizhou@kingsoft.com>

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: https://github.com/kubeedge/kubeedge/blob/master/CONTRIBUTING.md
```
Comment (llhuii) [inline .github/PULL_REQUEST_TEMPLATE.md:3]: Use Sedna's contributing link https://github.com/kubeedge/sedna/blob/main/CONTRIBUTING.md

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+<!--
+*Automatically closes linked issue when PR is merged.
+Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
+_If PR is about `failing-tests or flakes`, please post the related issues/tests in a comment and do not use `Fixes`_*
+-->
+Fixes #
+
+**Special notes for your reviewer**:
+
+**Does this PR introduce a user-facing change?**:
```
Comment (llhuii) [inline .github/PULL_REQUEST_TEMPLATE.md:37]: Look like Sedna no need the release-note-section

**Comment by llhuii** [comment]
pls squash two commits

**Comment by haiker2011** [comment]
@llhuii  already squash two commits

**Pull request merged**